custom_mappings.yaml

```diff
@@ -1,4 +1,174 @@
 entries:
+  - title: "Hyakusho Kizoku-the farmer’s days"
+    synonyms:
+      - "Hyakushou Kizoku"
+      - "百姓貴族"
+    seasons:
+      - season: 1
+        anilist-id: 155971
+      - season: 2
+        anilist-id: 171689
+  - title: "Bananya"
+    synonyms:
+      - "Bananya: Fushigi na Nakamatachi"
+      - "Bananya and the Curious Bunch"
+      - "ばなにゃ ふしぎななかまたち"
+    seasons:
+      - season: 1
+        anilist-id: 110881
+      - season: 2
+        anilist-id: 21704
+      - season: 3
+        anilist-id: 180599
+  - title: "Thunderbolt Fantasy"
+    synonyms:
+      - "Thunderbolt Fantasy: Touriken Yuuki"
+      - "Thunderbolt Fantasy: Sword Travels in the East"
+      - "Thunderbolt Fantasy -東離劍遊紀-"
+    seasons:
+      - season: 1
+        anilist-id: 98143
+      - season: 2
+        anilist-id: 99557
+      - season: 3
+        anilist-id: 119806
+      - season: 4
+        anilist-id: 138522
+  - title: "The Idolm@ster Shiny Colors"
+    synonyms:
+      - "The iDOLM@STER: Shiny Colors"
+      - "アイドルマスター シャイニーカラーズ"
+    seasons:
+      - season: 1
+        anilist-id: 162889
+      - season: 2
+        anilist-id: 175533
+  - title: "Sword Art Online Alternative: Gun Gale Online"
+    synonyms:
+      - "ソードアート・オンライン オルタナティブ ガンゲイル・オンライン"
+    seasons:
+      - season: 1
+        anilist-id: 100183
+      - season: 2
+        anilist-id: 167141
+  - title: "The New Prince of Tennis"
+    synonyms:
+      - "Shin Tennis no Ouji-sama"
+      - "新テニスの王子様"
+    seasons:
+      - season: 1
+        anilist-id: 11371
+      - season: 2
+        anilist-id: 20941
+      - season: 3
+        anilist-id: 140187
+      - season: 4
+        anilist-id: 165810
+  - title: "Shangri-La Frontier"
+    synonyms:
+      - "シャングリラ・フロンティア"
+    seasons:
+      - season: 1
+        anilist-id: 151970
+      - season: 2
+        anilist-id: 176508
+  - title: "Seirei Gensouki: Spirit Chronicles"
+    synonyms:
+      - "Seirei Gensouki"
+      - "精霊幻想記"
+    seasons:
+      - season: 1
+        anilist-id: 126546
+      - season: 2
+        anilist-id: 141182
+  - title: "Rurouni Kenshin (2023)"
+    synonyms:
+      - "Rurouni Kenshin: Meiji Kenkaku Romantan (2023)"
+      - "るろうに剣心 －明治剣客浪漫譚－(2023)"
+    seasons:
+      - season: 1
+        anilist-id: 142877
+      - season: 2
+        anilist-id: 171637
+  - title: "Tonbo!"
+    synonyms:
+      - "Ooi! Tonbo"
+      - "オーイ! とんぼ"
+    seasons:
+      - season: 1
+        anilist-id: 164440
+      - season: 2
+        anilist-id: 178729
```

Comment:
In Tonbo, I think second season should be listed as `season: 1` with `start: 14`.
TVDB only shows 1 season including all 26 episodes from the 2 AniList entries

Reply:
Oh looks like they changed this 2 days ago 😭
Fixed.

Reply:
I think you missed changing the season from `2` to `1`